### ISSUE TITLE
set up communication channel for development and usage discussions: gitter.im or mailing list

### ISSUE THREAD

**Issue by gauteh** [odeke-em/drive#518]
It would be great with somewhere to discuss patches or changes etc that are not pull-request ready. Perhaps to do quick guiding while discussing how to implement a patch for a new user or to ask for some guidance on usage, or on whether something actually is a bug.

A persistent / searchable place would probably be the best. One such service is gitter.im which is well-integrated with GitHub (and can also be accessed from IRC). But anything (like a standard IRC channel or a mailinglist) would be good!

ref #514.

**Comment by clawoflight** [comment]
Mailing lists are great because they are asynchronous, i.e. everyone can handle messages when he has time.

**Comment by gauteh** [comment]
True; since this is a google thing a google-groups mailing list might be appropriate. On the other hand I also like the way gitter/irc encourages quick replies and low-threshold discussions.

**Comment by odeke-em** [comment]
Aye aye, we'll look into setting those up. I'll just wait for other users to chime in.
Just to reiterate, I prefer users dropping issues because I've never managed a mailing list or an IRC but I see the utility there, users dropping in at their own time, plus there is always a first time for everything. Problem though is that information is spread out over multiple areas and search history is lost unless you serach through the repo, the wiki and the other sources.

**Comment by CountryBumkin** [comment]
I think the existing arrangements are adequate. If people make a Github user name they can put an email 'watch' on issues relevant to their interests and every update is emailed with a link back to the topic if they want to comment.

IRC is of limited use with users in different time zones and work hours around the world.

Searching is easy and specific if a Google search is framed like this example ...
site:github.com/odeke-em/drive rate error

Regards CB

**Comment by odeke-em** [comment]
@CountryBumkin well presented -- you mentioned exactly what I had in mind but hadn't properly described, thanks!

**Comment by canpolat** [comment]
I agree with @CountryBumkin. Keeping the communication in a single channel looks tidier to me (especially for a not-so-big community like this).

**Comment by gauteh** [comment]
This all makes sense. GitHub-issues does a better job than a
mailing-list, and it is a good argument to keep the communication to a
single channel. 

Note that gitter.im content is persistent and available whenever you log
on, other than that it has a flat structure and integrates nicely with
issue activity and other github activity (commens on code, commits,
etc.. ).

Regards, Gaute

**Comment by odeke-em** [comment]
@canpolat, you nailed that right on the head, yap the drive community is very small and the issues here seem sufficient. @gauteh if the community were big and questions couldn't be resolved real quick this would work well, but I'll leave this open for a bit more time for other people to chime in.

**Comment by gauteh** [comment]
That sounds good!

Emmanuel T Odeke writes on December 19, 2015 10:24:

**Comment by aerth** [comment]
Glad to see this issue come up! I had no idea about irc.gitter.im and I am stoked
